understanding/22/consistent-help.html

```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
 
-        <p>The human contact details enable users to connect with the  organization or the part of the organization that can assist with the content. For example, an online jobs / recruitment portal may provide a contact method for the team that supports the recruitment portal and not a catch-all for the entire company. Each layer of contact added prolongs the time before the user will receive help.</p>
+            <p>For pages for which no human support is available it helps if a self-help option says that no human support is available. Self-help options can go beyond allowing the user to search within the site. Contextual help is still recommended (see <a href="help">Success Criterion 3.3.5</a> for more information) but a self-help option provides a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.</p>
 
-        <p>The human contact mechanism enables a person to express what they are looking for using their own words. For some with cognitive disabilities, this may be the best way for them to find an answer to their problem.</p>
+            <p>Chatbots can work for many people, and particularly for people with cognitive disabilities if they:</p>
 
-        <p>For pages for which no human support is available, it is best if a self-help option says that no human support is available.  Self-help options can go beyond allowing the user to search within the site. Contextual help is still recommended (See Success Criterion 3.3.5 for more information) but a self-help option provides a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.</p>
+            <ul>
+               <li>recognize misspelled words,</li>
+               <li>provide human contact details if the chatbot is unable to provide a satisfactory response after 3 attempts, and</li>
+               <li>can be dismissed with a single interaction, and recalled using a link or button.</li>
+            </ul>
 
-        <p>Chatbots can work more effectively for everyone, and particularly for people with cognitive disabilities if they:</p>
+            <p>This criterion does not require that a site provide a help mechanism. However, when help is available:</p>
+            <ul>
+               <li>people who may have difficulty locating help are more likely to find it and complete their task.</li>
```

Comment:
ed: inconsistent capitalization (this and the following line should probably be capitalized as L102 is)